docs/quickstart/multicluster.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Multicluster setup
+weight: 200
+---
+
+The following procedure shows you how to collect metrics from a peer cluster to an existing One Eye deployment. The cluster running the One Eye deployment is called the **observer cluster**, the new cluster you are collecting metrics from is called the **peer cluster**. For details on how One Eye collects metrics from multiple clusters, see {{% xref "/docs/one-eye/multicluster/_index.md" %}}.
+
+## Prerequisites
+
+- You have completed the steps described in {{% xref "/docs/one-eye/quickstart/_index.md" %}} and you have a working One Eye deployment.
```

Comment:
we don't need the full quickstart here as the steps describe the full setup process.